dtmi/com/microchip/wfi32_iot_wm-1.json

```diff
@@ -0,0 +1,228 @@
+{
+  "@id": "dtmi:com:Microchip:WFI32_IoT_WM;1",
+  "@type": "Interface",
+  "contents": [
+    {
+      "@type": [
+        "Telemetry",
+        "NumberValue",
```

Comment:
What's the intent of  the co-type `NumberValue` ?

To be clear, this is a valid DTDL v2, I'm asking just to lean if there is a missing semantic type that we could schedule for v3.

Reply:
Hi Rido,

I am not sure because Daisuke created the device model - so you could ask him :)

I'm assuming that "NumberValue" is there to represent a generic type?

Thanks,
Randy

Reply:
I asked, and he does not recognize the unknown co-type `NumberValue`. 

If there is no reason to use it, I'd suggest removing it before merging this PR.

Reply:
I think I know what happened - I imported the JSON file into IoT Central, made a couple of modifications in the Device Template builder, and then exported to a new JSON file.  So I think IoT Central added the NumberValue types...

Anyways, can you please run the DTDL v2 test again on this latest commit? Thanks!!!